config/formats.js

```diff
@@ -960,7 +960,8 @@ exports.Formats = [
 		],
 
 		mod: 'gen5',
-		ruleset: ['Pokemon', 'Team Preview', 'Standard Ubers'],
+		ruleset: ['Pokemon', 'Team Preview', 'Standard'],
+		removedRules: ['Evasion Moves Clause'],
```

Comment:
<code>removedRules</code> is not a valid format property. It's only used in the tournaments code.

Reply:
Yeah I was gonna look into that but it must have slipped my mind. Direct me to the right file?

Reply:
There is no file to direct you to in this case. It's only used as a local variable in displaying custom rules for tournaments.

Reply:
I meant to the file where I can add in that functionality.

Reply:
<code>sim/dex.js</code> and <code>sim/dex-data.js</code>, but you should probably do that in a separate PR.

Reply:
I think this should be best done with:

`ruleset: ['Pokemon', 'Team Preview', 'Standard', '!Evasion Moves Clause'],`